pkg/cmd/util/oauth.go

```diff
@@ -0,0 +1,22 @@
+package util
```

Comment:
Move to `pkg/oauth/util/util.go` or maybe combine with `pkg/oauth/discovery/discovery.go`.